### PR TITLE
task was completed

### PR DESCRIPTION
**Pull request by Glebatikot**
- [DEMO LINK](https://Glebatikot.github.io/layout_calendar/)
- [TEST REPORT LINK](https://Glebatikot.github.io/layout_calendar/report/html_report/)

### CODE REVIEW

**Review by ivan-kadykalo — CHANGES_REQUESTED** [review]
Student review first ⚠️

File: src/index.html [hunk 1]
```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start--day--sun calendar--month-lenght--30">
```
Comment (ivan-kadykalo) [inline src/index.html]:
```suggestion
    <div class="calendar calendar--start--day--sun calendar--month-length--30">
```

**Review by ivan-kadykalo — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (Glebatikot) [reply src/index.html]: hahahha))

**Review by VETALEK — COMMENTED** [review]
You still need a student review 😉

**Comment by Yura-Lysenko** [comment]
looks good!!)

**Review by Yura-Lysenko — APPROVED** [review]

File: src/styles/normilize.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
+* {
+  margin: 0;
```
Comment (VETALEK) [inline src/styles/normilize.scss]: are you sure you need to reset all the margins?

Comment (VETALEK) [reply src/styles/normilize.scss]: you use only `<div>`s, they don't even have default `margin` 😭

**Review by VETALEK — CHANGES_REQUESTED** [review]
mind placing your calendar in the center 😉
and you need some space from the top

image

**Review by VETALEK — APPROVED** [review]
GJ 🔥

**Pull request closed without merging**